Reload nginx or stunnel when updating certificates

patrixl opened the pull request:
Instead of reloading the admin API.  Closes issue #15.

Commit:
Reload nginx or stunnel when updating certificates rather than reloading the admin API

Commit:
Fix typo in killall

Commit:
Redirect output and err to null, make sure a failed killall doesn't stop the script

aw reviewed (commented):
Works with `nginx` but not with `stunnel`

```
2016.10.01 10:03:16 LOG3[main]: error queue: 140DC002: error:140DC002:SSL routines:SSL_CTX_use_certificate_chain_file:system lib
2016.10.01 10:03:16 LOG3[main]: error queue: 20074002: error:20074002:BIO routines:FILE_CTRL:system lib
2016.10.01 10:03:16 LOG3[main]: SSL_CTX_use_certificate_chain_file: 2001002: error:02001002:system library:fopen:No such file or directory
2016.10.01 10:03:16 LOG3[main]: Service [https]: Failed to initialize SSL context
2016.10.01 10:03:16 LOG3[main]: Failed to reload the configuration file
```

aw commented:
`stunnel` doesn't HUP because the full path to `enterprise.pem` is not in `stunnel.conf`. [See here](https://github.com/unscramble/jidoteki-admin-api/blob/master/stunnel.conf#L4)

Implementing this change now would tightly couple `jidoteki-admin` with `jidoteki-admin-api`... not sure if I want that...

aw committed:
Add symlink for enterprise.pem

When `stunnel (3)` forks, its new `cwdir()` is `/`, so a simple symlink in that directory will allow a HUP to correctly reload the config.

aw commented:
Rebased and merged manually.
